python-3.10/Dockerfile

```diff
@@ -81,6 +79,8 @@ RUN mkdir /home/default \
         pandas \
         patsy \
         plotly \
+        pipdeptree \
+        prophet \
```

Comment:
These packages are new, `torch` was removed. Is it OK?

Reply:
torch wasn't removed, it's installed separately below.

`pipdeptree` is the tool I used to find out which package had the nvidia dependency, it is small and could be helpful in the future for that sort of debugging.

prophet is a very heavily used package by our users, and I think it is helpful to include it.  (it's included by default also in the base R)

Reply:
> prophet is a very heavily used package by our users

did anybody ask for it? is it somewhere in the task description? or do we just have the feeling it's usefull buth otherwise totally unrelated to this? :smile:

Reply:
3 :-)